sdcm/logcollector.py

```diff
@@ -62,6 +62,7 @@ class BaseLogEntity():  # pylint: disable=too-few-public-methods
     which require to be logged, stored locally, uploaded to
     S3 storage
     """
+    receive_entity_timeout = 300
```

Comment:
naming is a bit confusing across this PR
it introduce 4 now parameter that can be passed down:
* `receive_entity_timeout`
* `wait_collecting_timeout`
* `receive_timeout`
* `rsync_timeout`

Reply:
can we just call anything saved on the objects Collector Entity, `collect_timeout`

and change `receive_timeout` and `rsync_timeout`, to just `timeout` ?
* rsync is implementation detail of remoter.py that should be leaking out
* and `receive_timeout` on a call to function named `receive_log`, is a bit excessive isn't DRY.

Reply:
Thanks for clarification. Fixed